crud-operations/crud_sql_alchemy.py

```diff
@@ -0,0 +1,23 @@
+from sqlalchemy import create_engine, Column, Integer, String
+from sqlalchemy.orm import DeclarativeBase, sessionmaker
+
+
+class Base(DeclarativeBase):
+    pass
+
+
+class Bird(Base):
+    __tablename__ = "bird"
+    id = Column(Integer, primary_key=True)
+    name = Column(String)
+
+    def __repr__(self):
+        return f"<Bird(id={self.id}, name='{self.name}')>"
```

Comment:
:wrench: **Technical Recommendation**

Let's keep your code consistent with the tutorial:

```suggestion
        return f"Bird(id={self.id}, name={self.name!r})"
```